README.md

```diff
@@ -1 +1,143 @@
-# manifest-ledger
\ No newline at end of file
+<h1 align="center">Manifest Ledger</h1>
+
+<p align="center">
+  <a href="#overview"><img src="https://avatars.githubusercontent.com/u/90303796?s=200&v=4" alt="Lifted Initiative" width="100"/></a>
+</p>
+
+<p align="center">
+  <a href="https://codecov.io/gh/liftedinit/manifest-ledger">
+    <img src="https://codecov.io/gh/liftedinit/manifest-ledger/branch/reece/codecov/graph/badge.svg" alt="codecov"/>
+  </a>
+  <a href="https://goreportcard.com/report/github.com/liftedinit/manifest-ledger">
+    <img src="https://goreportcard.com/badge/github.com/liftedinit/manifest-ledger" alt="Go Report Card"/>
+  </a>
+  <a href="https://discord.gg/kQkaJzxvk9">
+    <img src="https://badgen.net/badge/icon/discord?icon=discord&label" alt="Discord"/>
+  </a>
+</p>
+
+## Overview
+
+The Manifest Network, built on the Cosmos SDK, is a blockchain tailored for decentralized AI infrastructure access. Initially employing a Proof of Authority (PoA) model it ensures a secure and efficient launch phase, with a trusted validator set managing consensus.
+
+While PoA offers immediate stability and control, the Manifest Network aspires for greater decentralization. The future roadmap includes evolving towards a Proof of Stake (PoS) network, utilizing the underlying CometBft consensus mechanism inherent in the Cosmos SDK.
+
+## Table of Contents
+
+- [System Requirements](#system-requirements)
+- [Installation](#install--run)
+- [Testing](#testing)
+- [Helper](#helper)
+- [Modules](./MODULE.md)
+- [Validators](./network/manifest-1/POST_GENESIS.md)
+- [Multi Sig Guide](./network/manifest-1/MULTI_SIG.md)
+- [Contributing](./CONTRIBUTING.md)
+- [Security/Bug Reporting](./SECURITY.md)
+
+## System Requirements
+
+**Minimal**
+
+- 4 GB RAM
+- 100 GB SSD
+- 3.2 x4 GHz CPU
+
+**Recommended**
+
+- 8 GB RAM
+- 100 GB NVME SSD
+- 4.2 GHz x6 CPU
+
+**Software Dependencies**
+
+1. The Go programming language - <https://go.dev/>
+2. Git distributed version control - <https://git-scm.com/>
+3. Docker - <https://www.docker.com/get-started/>
+4. GNU Make - <https://www.gnu.org/software/make/>
+
+**Operating System**
+
+- Linux (x86_64) or Linux (amd64) Recommended Arch Linux
```

Comment:
Isn't `x86_64` and `amd64` the same thing?

Reply:
yea, BSD systems use amd64 and macOS uses x86_64. But yea they are pretty much the same thing. I assume this is a typo for arm64 (which we also support)

Reply:
changed to arm64